### DISCUSSION TITLE
Can we redirect user to our application instead of keycloak, if user won't give his consent to other IDP

### DISCUSSION THREAD

**Discussion by Sheeren-Caw** [keycloak/keycloak#11982] [Q&A]
I have added github as idp in keycloak, and in my github oauth app I have added keycloak redirect_uri as callback url. 

I am redirecting user to keycloak, which internally redirects user to git hub login page.  After login, on consent screen if user gives his consent, then github is redirecting back to keycloak with code in query parameter, which internally redirects back to my callback endpoint.
{base_url}/auth/realms/cdefense/broker/github/endpoint?code={code}&state={state}

However, if user doesn't give his access, then github is redirecting to keycloak with error.
{base_url}/auth/realms/cdefense/broker/github/endpoint?error=access_denied&error_description=The+user+has+denied+your+application+access.&error_uri=https%3A%2F%2Fdocs.github.com%2Fapps%2Fmanaging-oauth-apps%2Ftroubleshooting-authorization-request-errors%2F%23access-denied&state={state}

Is there a way I can redirect user back to my application from keycloak if user denied access.

**Comment by kartikcldfnse** [comment]
I am also facing same issue.

Basically, how do we redirect the user to my application if the user decides to cancel the process and not give us permission?
